pkg/phlaredb/tsdb/index/index.go

```diff
@@ -279,14 +283,14 @@ type FileWriter struct {
 	name string
 }
 
-func NewFileWriter(name string) (*FileWriter, error) {
+func NewFileWriter(name string, bufferSize int) (*FileWriter, error) {
```

Comment:
I will revert these changes in tsdb/index/index.go - it is not used anymore

Reply:
It's currently used in compactor. I see no need to change it now